src/extras/simplePagination.ts

```diff
@@ -0,0 +1,83 @@
+import { stringifyVariables } from 'urql';
+import { Resolver, Variables, NullArray } from '../types';
+
+export interface PaginationParams {
+  offsetArgument?: string;
+  limitArgument?: string;
+}
+
+export const simplePagination = ({
+  offsetArgument = 'from',
```

Comment:
Is this the correct default?

Reply:
I think `from` is pretty common for cursors, but if I'm not mistaken `skip` is more common for offset-based pagination, i.e. `skip`/`limit`